lib/msal-core/src/utils/StringUtils.ts

```diff
@@ -15,4 +15,18 @@ export class StringUtils {
     static isEmpty(str: string): boolean {
         return (typeof str === "undefined" || !str || 0 === str.length);
     }
+
+    /**
+     * Check if a string's value is a valid JSON object
+     *
+     * @param str
+     */
+    static isValidJson(str: string): boolean {
+        try {
+            const parsedValue = JSON.parse(str);
+            return (parsedValue && typeof parsedValue === "object");
```

Comment:
Can you please add a comment as to why this check is needed?

Reply:
Will do